webclient/js/elems/raflow.js

```diff
@@ -2128,13 +2157,31 @@ window.loadRABGInfoForm = function () {
                                         "PriorAddress", "PriorLandLoardName", "PriorLandLoardPhoneNo",
                                         "PriorLengthOfResidency", "PriorReasonForMoving"];
 
+                                    // These all fields are not required when transanctant is only user
+                                    var listOfNotRequiredFields = ["SSN", "TelephoneNo",
+                                        "Phone", "EmailAddress", "Position",
+                                        "GrossWages"];
+
                                     if(data.status === 'success'){
                                         var record = data.record; // record from the server response
                                         // w2ui.RABGInfoForm.record = data.record.Data;
-                                        if(raBGInfoGridRecord.IsUser && !raBGInfoGridRecord.IsPayor && !raBGInfoGridRecord.IsGurantor){
+
+                                        // Set the form tile
+                                        w2ui.RABGInfoForm.header = 'Background Information - ' + record.FirstName + ' ' + record.MiddleName + ' ' + record.LastName;
```

Comment:
Please, fix this for company name also if IsCompany is enabled!

Reply:
This work is in progress.